src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/media/index.js

```diff
@@ -38,7 +41,8 @@ const generateImageBlock = block => {
           type: 'rawImage',
           model: {
             copyrightHolder: block.imageCopyright,
-            locator: block.imageUrl,
+            locator: `https://${block.imageUrl.replace('$recipe', '1024x576')}`,
```

Comment:
Just wondering where these dimensions come from/equate to?

Reply:
These are the PAL HD dimensions - replicated from here: https://github.com/bbc/simorgh/blob/b3409e61bb773de8526eab4e178b076ec98fc7b0/src/app/containers/MediaPlayer/helpers/metadata/index.js#L3